File: backend/src/main/java/com/trimix/challenge/personas/dto/PersonaDto.java
package com.trimix.challenge.personas.dto;

import java.sql.Date;

import com.trimix.challenge.personas.enums.TipoDocumentoEnum;

import lombok.Data;

@Data
public class PersonaDto {
    private long id;
    private String nombre;
    private String apellido;
    private Date fechaNacimiento;
    private String numeroDocumento;
    private TipoDocumentoEnum tipoDocumento;
}

File: frontend/src/personas/types/PersonaDto.ts
import { TipoDocumentoEnumDto } from "./TipoDocumentoTypes";

interface PersonaDto {
  id?: number;
  nombre: string;
  apellido: string;
  fechaNacimiento: string;
  numeroDocumento: string;
  tipoDocumento: TipoDocumentoEnumDto;
}

export default PersonaDto;
